minor build updates 2023-01

brody2consult opened the pull request:
~~includes android-database-sqlcipher 4.5.3 update from PR #42~~

---

__UPDATED STATUS:__

~~I would now favor this over the major updates proposed in PR #44~~

_I would now favor the major updates proposed in PR #44._

brodybits commented:
I think it would be ideal to close this in favor of the major updates proposed in PR #44, hope one of these proposals will be considered to keep this test suite updated.

brodybits commented:
Looks like e113fa26f206a78a2780a46d555686b99b617a08 would be overridden by 2cf379d60579c619e61e9923ee08a5c106a2055a on master ... I would like to rebase the other changes

Commit:
update com.android.tools.build:gradle -> 4.2.2

ref:

- https://mvnrepository.com/artifact/com.android.tools.build/gradle?repo=google

Commit:
update androidx.sqlite:sqlite -> 2.1.0

working OK with Android SDK 30

ref:

- https://mvnrepository.com/artifact/androidx.sqlite/sqlite

Commit:
update Gradle wrapper -> 6.9.3

./gradlew wrapper --gradle-version 6.9.3

- update wrapper properties
- update wrapper JAR
- update wrapper scripts

ref:

- https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper
- https://gradle.org/releases/

brodybits commented:
I am now closing this in favor of the major build updates I proposed in PR #44, don't think #44 will break anything now that we are using SDK 33.
